packages/web-app-files/src/index.js

```diff
@@ -194,8 +194,11 @@ const routes = [
     children: [
       {
         name: 'personal',
-        path: 'all/:item?/:page?',
+        path: 'all/:item*',
         component: Personal,
+        transform: () => {
+          console.log('111')
```

Comment:
Trololo 👹

Reply:
👾👾👾 its just there to check if the reviews are propper 🤣  i can remove it now